docs/debian/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM debian:jessie
+MAINTAINER Andrey Ivanov stayhardordie@gmail.com
+
+ENV REPOSITORY=https://github.com/mapsme/omim.git \
+    DIR=/srv
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    wget \
+    git \
+    libssl-dev \
+    clang \
+    libc++-dev \
+    libglu1-mesa-dev \
+    libstdc++-4.8-dev \
+    qt5-default \
```

Comment:
Ubuntu is not Debian :)

> On Mar 31, 2016, at 11:34, mpimenov <notifications@github.com> wrote:
> 
> In docs/debian/Dockerfile <https://github.com/mapsme/omim/pull/2642#discussion_r58017469>:
> 
> > +
> > +ENV REPOSITORY=https://github.com/mapsme/omim.git \
> > +    DIR=/srv
> > +
> > +RUN apt-get update && \
> > +    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
> > +    build-essential \
> > +    ca-certificates \
> > +    wget \
> > +    git \
> > +    libssl-dev \
> > +    clang \
> > +    libc++-dev \
> > +    libglu1-mesa-dev \
> > +    libstdc++-4.8-dev \
> > +    qt5-default \
> Are you sure qt5-default is enough in all situations? The Ubuntu section above explains a tricker way to install qt.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2642/files/3ba974164ea2e51d06df9b1904230446b044226d#r58017469>